lib/services/utils.dart

```diff
@@ -5,6 +5,10 @@ import 'package:intl/intl.dart';
 import '../components/feature_badge.dart';
 import '../helpers/btv_colors.dart';
 
+class Globals {
+  static final utcOffset = _getUtcOffset();
```

Comment:
This can actually change during the lifetime of a session, because a lot of our users travel between timezones often (traveling to brunstad). So I think we should calculate this on-the-fly where it's needed. It's a cheap operation.